src/Aspire.Hosting/Utils/PasswordUtil.cs

```diff
@@ -6,4 +6,47 @@ namespace Aspire.Hosting.Utils;
 internal static class PasswordUtil
 {
     internal static string EscapePassword(string password) => password.Replace("\"", "\"\"");
+
+    /// <summary>
+    /// Returns a random password of length <paramref name="length"/> that does not contain any characters that would be escaped by <see cref="EscapePassword(string)"/>.
+    /// </summary>
+    /// <remarks>If <paramref name="length"/> is greater than or equal to four, the password is guaranteed to contain an uppercase ASCII letter, a lowercase ASCII letter, a digit, and a symbol.</remarks>
+    internal static string GeneratePassword(int length = 20)
+    {
+        return string.Create(length, 0, (buffer, _) =>
```

Comment:
I think this password generation might be a problematic. Imagine that we set a password length of 6. We would already know that the first character is going to be in the range A-Z, second a-z, third 0-9, and forth one of the specials. Obviously, that becomes less of an issue the longer you get.